.travis.yml

```diff
@@ -41,7 +49,7 @@ before_script:
 script:
   - flake8 --exclude migrations,opentreemap/settings/local_settings.py,*.pyc opentreemap
   - npm run check
-  - testem ci -l Firefox
```

Comment:
It wasn't my intention, but there isn't really a need to have the `-l Firefox` anyways.

It _was_ there because we had issues w/ PhantomJS on Travis hanging indefinitely, but that seems like it is no longer an issue.